.clineworkflows/review.md

```
@@ -0,0 +1,274 @@
+          - "Check algorithm against published sources (search Ref MCP or Tavily MCP)"
+          - "Verify mathematical formulas match documentation"
+          - "Check for known indicator variants (e.g., Wilder's RSI vs. Cutler's RSI)"
+          - "Document any deviations from standard implementation"
+
+      - step: "9. Edge Cases & Error Handling"
+        tasks:
+          - "Division by zero scenarios tested"
+          - "Empty input handling"
+          - "Single value input"
+          - "All NaN input"
+          - "Period = 1 edge case"
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 4; Actual: 10

(MD007, ul-indent)